### ISSUE TITLE
Officially release Basque course

### ISSUE THREAD

**Issue by kantord** [LibreLingo/LibreLingo#2386]
Todo list:

- [ ] Find a list of places to announce the official release in in order to draw some initial attention to the course
- [ ] Link the Basque course on the home page
- [ ] Fix registration bug: https://github.com/LibreLingo/LibreLingo/issues/2299
- [ ] Create some community for the Basque course on Matrix or somewhere else

**Comment by kantord** [comment]
@RGBradley do you have some website or other contact link that you want to advertise in the places where we announce the launch? I think it could be useful

**Comment by kantord** [comment]
Here's a list of places where I think it would be a good idea to make an announcement:

## Reddit

- https://www.reddit.com/r/basque/
- https://www.reddit.com/r/basquecountry/
- https://www.reddit.com/r/spain/
- https://www.reddit.com/r/linguistics/ (not so sure about this one, but it's not among the list of not allowed content)
- https://www.reddit.com/r/languagelearning/
- https://www.reddit.com/r/coolgithubprojects/ (announce the GitHub repo, not the course itself)

**Comment by kantord** [comment]
@RGBradley do you perhaps know some more places where the announcement could be made? Perhaps there are some forums and websites for the Basque learning/teaching community?

**Comment by Lamdarer** [comment]
There probably is some sort of (Inter-/National) Basque Community or some kind of "Central Council of Basque Speakers", a Basque university or Basque Country Government education institution which could be kindly informed about it. 
If it hits a nerve there might even be some professional Basque speakers taking a look at the course and Basque ppl who find it useful might recommend it to their Basque learning friends etc.

**Comment by kantord** [comment]
> There probably is some sort of (Inter-/National) Basque Community or some kind of "Central Council of Basque Speakers", a Basque university or Basque Country Government education institution which could be kindly informed about it. If it hits a nerve there might even be some professional Basque speakers taking a look at the course and Basque ppl who find it useful might recommend it to their Basque learning friends etc.

yes, that's a good idea and I agree. I wonder if it's a good idea to release this thing in stages though. Perhaps release in stages? I'm thinking it's probably also not the best idea to post it all over reddit just in one day. What do you think?

**Comment by RGBradley** [comment]
As I mentioned in the email, the only place I can think of is https://buber.net/Basque/ which has a wide following. The main problem I see is getting feedback: suggestions, corrections, etc. I can also see if it gets a lot of users right off, I will be quickly overwhelmed, so probably better to let it grow slowly. As for the comment about the readme file  having links for contacts, would that be an email or to this page? 
rgb

**Comment by kantord** [comment]
>  so probably better to let it grow slowly.

that's probably a good idea.

What would be your plan to release it then?

**Comment by kantord** [comment]
> As for the comment about the readme file having links for contacts, would that be an email or to this page?

If you are comfortable to add your email, Twitter or some other way people can contact you personally that's great. Apart from that I would basically add links to the LibreLingo repo itself and some explanation on how to interact with the course and such.

**Comment by RGBradley** [comment]
I have an account DuoBasque@gmail.com which can be used for the course  to get feedback
maybe we could put in the introduction or if you could have messages sent to it from a comments section for the course

**Comment by RGBradley** [comment]
I don't have a reddit account, so could you do the announcement?

**Comment by kantord** [comment]
> I don't have a reddit account, so could you do the announcement?

Yes of course, that's not a problem! :+1:

**Comment by kantord** [comment]
> I have an account [DuoBasque@gmail.com](mailto:DuoBasque@gmail.com) which can be used for the course to get feedback maybe we could put in the introduction or if you could have messages sent to it from a comments section for the course

sure, I think that should work

**Comment by RGBradley** [comment]
Hi Kantord
I see the registration is still not working. Hope it gets fixed soon so users can make comments.
Also see that Basque is still not public.
Has there been any work on using the Omnilingo speech system in LibreLingo? Spanish pronunciation is close enough that it could be used for Basque. Purists would complain, but most Basque speaker in Spain use Spanish pronunciation.
best wishes
rick

**Comment by kantord** [comment]
>I see the registration is still not working. Hope it gets fixed soon so users can make comments.

This is not related to users ability to make comments, right? I mean, for now, feedback is not a logged in feature

> Also see that Basque is still not public.

It is not, should I make it public immediately? That takes almost no work, I mean, I can just add a button for that

> Has there been any work on using the Omnilingo speech system in LibreLingo? Spanish pronunciation is close enough that it could be used for Basque. 

Omnilingo? That's not meant to be integrated into LibreLingo, but there is a similar system built in. However, in order to make it work for Basque we should do this task: https://github.com/LibreLingo/LibreLingo/issues/714
